src/client/containers/AddNewRoom/AddNewRoom.js

```diff
@@ -1,40 +1,76 @@
 import React, { useState } from 'react';
-
 import RoomForm from '../../components/AddNewRoom/RoomForm';
-import fetchWithAuth from '../../utils/fetchWithAuth';
+import '../../components/AddNewRoom/AddNewRoomStyle.css';
+import PropTypes from 'prop-types';
 
-const AddNewRoom = () => {
+const AddNewRoom = ({ addUsers }) => {
```

Comment:
The suggested name is `addedUsers`. `addUsers` reads more like `add users` which is a command, not an array.